Documentation/config.adoc

```diff
@@ -0,0 +1,252 @@
+
+// NOTE: The contents of this file are automatically generated from source code comments.
+// If you wish to make a change or an addition to the content in this document, do so by changing the code comments.
+
+= Config map fields for the Cluster Monitoring Operator 
+
+The following tables describe fields that you can use in config map objects for the Cluster Monitoring Operator. These fields enable you to do fine-grained configuration of platform monitoring and user workload monitoring.
+
+== ClusterMonitoringConfiguration
+
+
```

Comment:
I couldn't find `enableUserWorkload` in the generated doc., Am I missing something here?